web/tests/upgrade.test.js

```diff
@@ -187,10 +187,6 @@ run_test("autopay_form_fields", () => {
         document.querySelector("#autopay-form #automatic_license_count").value,
         "{{ seat_count }}",
     );
-    assert.equal(
-        document.querySelector("#autopay-form #manual_license_count").min,
-        "{{ seat_count }}",
-    );
```

Comment:
Can this somehow be tested? If seems that gating `min` inside an if in the template: `{% if not exempt_from_license_number_check %}min="{{ seat_count }}"{% endif %}` is what breaks this. I tried searching around a bit but didn't find instances of these tests testing stuff inside `ifs` like that

Reply:
I don't think so -- this test seems to be hackily reading the HTML of the page, rather than actually using the server.

If we cared, we could make this a puppeteer test, but I don't think we do.